sw/device/tests/sim_dv/otp_ctrl_hw_cfg_test.c

```diff
@@ -0,0 +1,97 @@
+// Copyright lowRISC contributors.
```

Comment:
Move this test to `sw/device/tests` because it can be run on Verilator / FPGA also.